src/box/iproto.cc

```diff
@@ -2987,8 +2996,9 @@ iproto_send_listen_msg(struct evio_service *binary)
 int
 iproto_listen(const char *uri)
 {
-	if (iproto_send_stop_msg() != 0)
-		return -1;
+	struct evio_service tmp;
+	memset(&tmp, 0, sizeof(tmp));
```

Comment:
This doesn't look like a proper initialization of evio_service. Shouldn't you use evio_service_init?

Reply:
Moved this part to lua